pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -144,7 +148,14 @@ func (exp *flowExporter) Export(collectorAddr string, collectorProto string, sto
 				}
 				// Build and send flow records to IPFIX collector.
 				exp.flowRecords.BuildFlowRecords()
-				err := exp.sendFlowRecords()
+				var dataSet ipfix.IPFIXSet
+				if exp.v4Enabled {
+					dataSet = ipfix.NewSet(ipfixentities.Data, exp.templateIDv4, false)
+				} else if exp.v6Enabled {
+					dataSet = ipfix.NewSet(ipfixentities.Data, exp.templateIDv6, false)
+				}
```

Comment:
We should create two data sets each for ipv4 and ipv6 and send them as args to sendFlowRecord. Depending on the flow record type, that function can pick the appropriate dataset.
In the case of dual stack, both IPv4 and IPv6 records are present.

Reply:
Got it. Updated. Thanks!